Week01/Problem03/msoto_03.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+##What is the largest prime factor of the number 600851475143?
+
+i=2
+number=600851475143
+while i < number:
+	while number%i == 0:
+		number = number/i
```

Comment:
You should be doing floor division, here, i.e., rounding down the division to the nearest integer.  Otherwise, you may end up with a decimal (float) value for `number`.  So, this line could be replaced with
```
number = number // i
```
or, equivalently,
```
number //= i
```
where the `//` operator corresponds to floor division.